packages/editor/vite.config.ts

```diff
@@ -3,11 +3,7 @@ import { defineConfig } from 'vitest/config';
 
 export default defineConfig(({ mode }) => ({
   plugins: [sveltekit()],
-  server: {
-    fs: {
-      allow: ['prime.css'],
-    },
-  },
+  server: {},
```

Comment:
Can we remove this whole line since it's unused?

Reply:
Will do -- kept for reference but will do

Reply:
Done.